app/views/admins/contacts/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= simple_form_for(@contact, url: path) do | form | %>
+
+  <%= form.error_notification %>
+
+  <div class="form-row">
+    <div class="col-sm-12 col-md-6">
+      <%= form.input :name, required: true, autofocus: true %>
+      <%= form.input :email, required: true %>
+    </div>
+
+    <div class="col-sm-12 col-md-6">
+      <%= form.input :phone,  required: false %>
```

Comment:
seria legal ter máscara de telefone, tanto pra 8 quanto pra 9 dígitos

Reply:
Só usando js, como outras PR teriam também outros mascaras, achamos melhor manter sem pq poderíamos ter mais de uma lib, (adicionada em cada PR) para resolver o mesmo problema.

Reply:
Use the github.com/igorescobar/jQuery-Mask-Plugin, no gem is required. This is used by tabler.io